cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Please remove these unused fixtures. We can always add the relevant ones in the future.